### PR TITLE
Bot API 5.0

### PR DESCRIPTION
**Pull request by Olegt0rr**
# Description

Updated Telegram Bot API 5.0

## Type of change
- [ ] Documentation (typos, code examples or any documentation update)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: aiogram/bot/bot.py [hunk 1]
```diff
@@ -176,6 +176,24 @@ async def get_me(self) -> types.User:
         result = await self.request(api.Methods.GET_ME, payload)
         return types.User(**result)
 
+    async def log_out(self) -> types.User:
```
Comment (uwinx) [inline aiogram/bot/bot.py:179]: return annotation should be false

Comment (Olegt0rr) [reply aiogram/bot/bot.py]: fixed 
https://github.com/aiogram/aiogram/pull/454/commits/ee523afb8c2c7ba18987d8f6982584d548e167e8

File: aiogram/bot/bot.py [hunk 2]
```diff
@@ -176,6 +177,43 @@ async def get_me(self) -> types.User:
...
+        Returns True on success. Requires no parameters.
+
+        Source: https://core.telegram.org/bots/api#close
+
+        :return: Returns True on success
+        :rtype: :obj:`base.Boolean`
+        """
+        payload = generate_payload(**locals())
+
+        result = await self.request(api.Methods.CLOSE, payload)
```
Comment (uwinx) [inline aiogram/bot/bot.py:214]: let's just return call result, without creating new name.

```python3
     return await self.request(api.Methods.CLOSE, payload)
```

same for .log_out

File: aiogram/bot/base.py [hunk 3]
```diff
@@ -173,6 +174,8 @@ def with_token(self, bot_token: base.String, validate_token: Optional[base.Boole
         finally:
             self._ctx_token.reset(token)
 
+    @deprecated("This method behaviour will be changed in aiogram v3.0. "
```
Comment (uwinx) [inline aiogram/bot/base.py:177]:
```suggestion
    @deprecated("This method's behavior will be changed in aiogram v3.0. "
```

biased "behavior" word change, anyways can keep it unchanged

File: aiogram/bot/bot.py (hunk 2 above)
Comment (Olegt0rr) [reply aiogram/bot/bot.py:242]: It's a code style of all bool return method (look at other methods). 
I agree that it's need to be refactored, but **all** of them :)

File: aiogram/types/message.py [hunk 4]
```diff
@@ -89,6 +91,7 @@ class Message(base.TelegramObject):
     successful_payment: SuccessfulPayment = fields.Field(base=SuccessfulPayment)
     connected_website: base.String = fields.Field()
     passport_data: PassportData = fields.Field(base=PassportData)
+    proximity_alert_triggered: typing.Optional[ProximityAlertTriggered] = fields.Field(base=ProximityAlertTriggered)
```
Comment (uburuntu) [inline aiogram/types/message.py:94]:
```suggestion
    proximity_alert_triggered: ProximityAlertTriggered = fields.Field(base=ProximityAlertTriggered)
```

File: aiogram/types/chat.py [hunk 5]
```diff
@@ -27,13 +28,16 @@ class Chat(base.TelegramObject):
     last_name: base.String = fields.Field()
     all_members_are_administrators: base.Boolean = fields.Field()
     photo: ChatPhoto = fields.Field(base=ChatPhoto)
+    bio: typing.Optional[base.String] = fields.Field()
```
Comment (uburuntu) [inline aiogram/types/chat.py:31]:
```suggestion
    bio: base.String = fields.Field()
```

File: aiogram/types/chat.py [hunk 6]
```diff
@@ -27,13 +28,16 @@ class Chat(base.TelegramObject):
...
     description: base.String = fields.Field()
     invite_link: base.String = fields.Field()
     pinned_message: 'Message' = fields.Field(base='Message')
     permissions: ChatPermissions = fields.Field(base=ChatPermissions)
     slow_mode_delay: base.Integer = fields.Field()
     sticker_set_name: base.String = fields.Field()
     can_set_sticker_set: base.Boolean = fields.Field()
+    linked_chat_id: typing.Optional[base.Integer] = fields.Field()
+    location: typing.Optional[ChatLocation] = fields.Field()
```
Comment (uburuntu) [inline aiogram/types/chat.py:40]:
```suggestion
    linked_chat_id: base.Integer = fields.Field()
    location: ChatLocation = fields.Field()
```

**Review by JrooTJunior — APPROVED** [review]

File: aiogram/types/message.py [hunk 7]
```diff
@@ -310,6 +311,10 @@ async def answer(
             fixed-width text or inline URLs in your bot's message.
         :type parse_mode: :obj:`typing.Optional[base.String]`
 
+        :param entities: List of special entities that appear in message text,
+            which can be specified instead of parse_mode
+        :type entities: :obj:`typing.Optional[typing.List[types.MessageEntity]]`
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 8]
```diff
@@ -1363,6 +1370,10 @@ async def reply(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 9]
```diff
@@ -379,6 +380,10 @@ async def answer_photo(
             fixed-width text or inline URLs in your bot's message.
         :type parse_mode: :obj:`typing.Optional[base.String]`
 
+        :param caption_entities: List of special entities that appear in message text,
+            which can be specified instead of parse_mode
+        :type caption_entities: :obj:`typing.Optional[typing.List[types.MessageEntity]]`
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 10]
```diff
@@ -1438,6 +1445,10 @@ async def reply_photo(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 11]
```diff
@@ -710,6 +711,10 @@ async def answer_video(
             fixed-width text or inline URLs in the media caption
         :type parse_mode: :obj:`typing.Optional[base.String]`
 
+        :param caption_entities: List of special entities that appear in message text,
+            which can be specified instead of parse_mode
+        :type caption_entities: :obj:`typing.Optional[typing.List[types.MessageEntity]]`
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 12]
```diff
@@ -1775,6 +1782,10 @@ async def reply_video(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 13]
```diff
@@ -551,6 +552,10 @@ async def answer_animation(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 14]
```diff
@@ -1622,6 +1629,10 @@ async def reply_animation(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 15]
```diff
@@ -452,6 +453,10 @@ async def answer_audio(
             fixed-width text or inline URLs in your bot's message.
         :type parse_mode: :obj:`typing.Optional[base.String]`
 
+        :param caption_entities: List of special entities that appear in message text,
+            which can be specified instead of parse_mode
+        :type caption_entities: :obj:`typing.Optional[typing.List[types.MessageEntity]]`
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 16]
```diff
@@ -1529,6 +1536,10 @@ async def reply_audio(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 17]
```diff
@@ -639,6 +640,10 @@ async def answer_document(
             bold, italic, fixed-width text or inline URLs in your bot's message.
         :type parse_mode: :obj:`typing.Optional[base.String]`
 
+        :param caption_entities: List of special entities that appear in message text,
+            which can be specified instead of parse_mode
+        :type caption_entities: :obj:`typing.Optional[typing.List[types.MessageEntity]]`
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

File: aiogram/types/message.py [hunk 18]
```diff
@@ -1722,6 +1729,10 @@ async def reply_document(
...
```
Comment (Olegt0rr) [inline aiogram/types/message.py]: исправил тип в https://github.com/aiogram/aiogram/pull/454/commits/cfa39c96d399df788cf09b570273d498c6ea87be

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**